build.gradle

```diff
@@ -37,5 +34,27 @@ subprojects { subproject ->
 
     compileJava compileOptions
     compileTestJava compileOptions
+
+    dependencies {
+      compileOnly group: 'io.vertx', name: 'vertx-codegen', version: '3.5.0'
```

Comment:
use knotx-dependencies pom here to get right versions. see how to do it https://docs.gradle.org/4.6/userguide/managing_transitive_dependencies.html?_ga=2.63949359.1281256360.1523425727-158124660.1522940345#sec:bom_import